src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/response/IdamRolesMappingResponse.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.cwrdapi.controllers.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class IdamRolesMappingResponse {
```

Comment:
Can we use Builder and build immutable response. if possible

Reply:
done